package.json

```diff
@@ -5,18 +5,18 @@
   "repository": "vega/vega",
   "scripts": {
     "build": "lerna run build && yarn docsbuild",
+    "build-editor-preview": "scripts/build-editor-preview.sh",
     "clean": "lerna clean --yes && lerna exec -- rimraf build && lerna exec -- rimraf LICENSE && rimraf node_modules yarn.lock",
     "data": "rsync -r node_modules/vega-datasets/data/* docs/data",
     "docs": "cd docs && bundle exec jekyll serve -I -l",
-    "docsbuild": "cd packages/vega/build && cp vega.js vega.min.js* vega-core.js vega-core.min.js* vega-schema.json ../../../docs/",
+    "docsbuild": "yarn data && cd packages/vega/build && cp vega.js vega.min.js* vega-core.js vega-core.min.js* vega-schema.json ../../../docs/",
     "license": "lerna exec -- cp ../../LICENSE .",
     "release": "yarn license && lerna publish from-package",
     "serve": "serve packages/vega/ -l 8080",
     "test": "yarn test:no-lint && yarn lint",
     "test:no-lint": "lerna run test --ignore vega-typings && lerna run test --scope vega-typings",
     "lint": "ESLINT_USE_FLAT_CONFIG=true yarn eslint -c eslint.config.mjs .",
-    "format": "yarn lint --fix",
-    "postinstall": "yarn data"
```

Comment:
If we must keep this as a postinstall step and can't mopve this to docsbuild, an alternate approach would be to replace this with a guard so that I can disable the postinstall step with an env variable.

Reply:
Convinced myself it would be less of a change for existing devs to take this route.

https://github.com/vega/vega/pull/3925/commits/d4bacf02452ea28f356da5edbed7e18f6d2bf8cd